packages/grid/src/vaadin-grid-scroll-mixin.js

```diff
@@ -136,6 +136,22 @@ export const ScrollMixin = (superClass) =>
         overflow += ' top';
       }
 
+      const scrollLeft = this.__getNormalizedScrollLeft(table);
+      if (scrollLeft > 0) {
+        overflow += ' start';
+      }
+
+      if (scrollLeft < table.scrollWidth - table.clientWidth) {
+        overflow += ' end';
+      }
+
+      if (this.__isRTL) {
+        overflow = overflow.replace(/start|end/gi, (matched) => {
+          return matched === 'start' ? 'end' : 'start';
+        });
+      }
+
+      // TODO: Remove "right" and "left" values in the next major.
```

Comment:
I wonder if we should be more proactive in helping users upgrade their themes to use the new values? Is it possible for us to check the contents of grid style sheets, and check if they use `overflow=left` or `overflow=right`, and log a warning in that case?

Reply:
>  Is it possible for us to check the contents of grid style sheets

In theory, we could add some logic to `registerStyles` but that sounds like extra overhead to me, especially for such a small change. IMO an entry in the upgrading guide in this case would be enough.